content/applications/finance/accounting/fiscal_localizations/localizations/egypt.rst

```diff
@@ -0,0 +1,244 @@
+=====
+Egypt
+=====
+
+.. _egypt/installation:
+
+Installation
+============
+
+:ref:`Install <general/install>` the following modules to get all the features of the Egyptian
+localization:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Technical name
+     - Description
+   * - :guilabel:`Egypt - Accounting`
+     - ``l10n_eg``
+     - Default :doc:`fiscal localization package <../overview/fiscal_localization_packages>`
+   * - :guilabel:`Egyptian E-invoice Integration`
+     - ``l10n_eg_edi_eta``
+     - :ref:`Egyptian Tax Authority (ETA) eInvoicing integration <egypt/einvoicing>`
+
+.. _egypt/einvoicing:
+
+Egyptian eInvoicing
+===================
+
+Odoo is compliant with the **Egyptian Tax Authority (ETA) eInvoicing** requirements.
+
+.. important::
+   Egyptian eInvoicing is available from Odoo 15.0. If needed, :doc:`upgrade
+   </administration/upgrade>` your database.
+
+.. seealso::
+   - `Video: Egypt E-invoicing <https://www.youtube.com/watch?v=NXuBPLR4pVw>`_
+   - :doc:`/administration/upgrade`
+
+.. _egypt/einvoicing-eta-portal:
+
+Register ERP on your ETA Portal
+-------------------------------
+
+You must register your Odoo ERP System on your ETA Portal to get your API credentials. You need
+these codes to :ref:`configure your Odoo Accounting app <egypt/einvoicing-configuration>`.
+
+Access your company profile on the ETA portal by clicking on :guilabel:`View Taxpayer Profile`.
+
+.. image:: egypt/taxpayer-profile.png
+   :align: center
+   :alt: Clicking on "View Taxpayer Profile" on an ETA invoicing portal
+
+Next, go to the :guilabel:`Representatives` section and then click on :guilabel:`Register ERP`.
+Fill out the :guilabel:`ERP Name` (e.g., ``Odoo``) and leave the other fields empty.
+
+.. image:: egypt/add-erp-system.png
+   :align: center
+   :alt: Filling out of the form to register an ERP system on the ETA portal.
+
+Once successfully registered, the website displays your API credentials:
+
+- Client ID
+- Client Secret 1
+- Client Secret 2
+
+.. note::
+   - ETA should give you a username and a password to access their online portal.
+   - Ask ETA to provide you with preproduction portal access as well.
+   - These codes are confidential and should be saved safely.
```

Comment:
I might be mistaken, but perhaps consider using "should be stored safely" instead of "saved safely".